cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -224,6 +228,9 @@ public void configure(Map<String, ?> configs) {
       reconnectBackoffMs = String.valueOf(DEFAULT_RECONNECT_BACKOFF_MS);
     }
 
+    _acceptableMetricRecordProduceDelayMs = ACCEPTABLE_NETWORK_DELAY_MS + new CruiseControlMetricsReporterConfig(configs, false)
+        .getLong(CruiseControlMetricsReporterConfig.CRUISE_CONTROL_METRICS_REPORTER_LINGER_MS_CONFIG);
```

Comment:
Should we add a config sanity check to ensure that `CRUISE_CONTROL_METRICS_REPORTER_LINGER_MS_CONFIG` cannot be larger than `CRUISE_CONTROL_METRICS_REPORTER_INTERVAL_MS_CONFIG`?